### DISCUSSION TITLE
Welcome to Franklin Discussions!

### DISCUSSION THREAD

**Discussion by oxagast** [oxagast/Franklin#3] [Announcements]
We use this place to discuss Franklin the GPT bot.

Discussion of feature requests start here!
If there is a recurring problem durring setup that you think may be a bug, discuss it here first!

* No flaming or talking shit to other users
* No harassment of any kind
* Only real feature requests please
* Do NOT open discussions or issues ONLY with things that Franklin has said about the bot itself.

**Comment by madmeech** [comment]
Just finishing reading your article in 2400. Good work & thank you for sharing! I think I will take a crack at getting Franklin running on an AWS instance. Owlbear wrote an article about creating a chatbot in a recent issue of Mag Pi. They used the Stanford Alpaca model in their example. I got it running on a RB PI 4 8GB, but it's not reliable at all and the build is already out of date. Still, it was a good learning experience and exercise, and I am happy with the results. Time to do the same with Franklin!
